doc/source/docker/tutorial-quickstart-docker-compose.rst

```diff
@@ -321,36 +309,71 @@ In ``with-tls.yml``, add the following:
         command:
           - --superlink
           - superlink:9092
-          - --supernode-address
+          - --clientappio-api-address
           - 0.0.0.0:9096
           - --isolation
           - process
           - --node-config
           - "partition-id=1 num-partitions=2"
+          - --ssl-ca-certfile=certificates/ca.crt
+          - --ssl-certfile=certificates/server.pem
+          - --ssl-keyfile=certificates/server.key
           - --root-certificates
-          - certificates/ca.crt
+          - certificates/superlink-ca.crt
         secrets:
           - source: superlink-ca-certfile
+            target: /app/certificates/superlink-ca.crt
+          - source: supernode-ca-certfile
             target: /app/certificates/ca.crt
+          - source: supernode-certfile
+            target: /app/certificates/server.pem
+          - source: supernode-keyfile
+            target: /app/certificates/server.key
+
+      clientapp-3:
+        command:
+          - --clientappio-api-address
+          - supernode-3:9096
+          - --root-certificates
+          - certificates/ca.crt
+        secrets:
+          - source: supernode-ca-certfile
+            target: /app/certificates/ca.crt
+
+Restart the services with:
+
+.. code-block:: bash
+
+    $ docker compose up --build -d
+    # or with TLS enabled
+    $ docker compose -f compose.yml -f with-tls.yml up --build -d
 
 Step 8: Persisting the SuperLink State and Enabling TLS
 -------------------------------------------------------
 
 To run Flower with persisted SuperLink state and enabled TLS, a slight change in the
 ``with-state.yml`` file is required:
 
-1. Comment out the lines 2-4 and uncomment the lines 5-9:
+1. Comment out the lines 2-8 and uncomment the lines 9-17:
 
    .. code-block:: yaml
        :caption: with-state.yml
        :linenos:
-       :emphasize-lines: 2-9
+       :emphasize-lines: 2-17
 
          superlink:
            # command:
-           #   - --insecure
-           #   - --database=state/state.db
+           # - --insecure
+           # - --executor
+           # - flwr.superexec.deployment:executor
+           # - --isolation
+           # - process
+           # - --database=state/state.db
            command:
+             - --executor
+             - flwr.superexec.deployment:executor
```

Comment:
Let's remove these lines since it's `flwr.superexec.deployment:executor` by default.